xml/abstract_commands.xml

```diff
@@ -94,4 +94,82 @@
         <field name="0" bits="23:0" />
     </register>
 
+    <register name="Access Memory">
```

Comment:
same note, can it be clarified that this is from the selected hart's perspective?